onadata/libs/utils/chart_tools.py

```diff
@@ -473,7 +473,7 @@ def get_chart_data_for_field(field_name,
     except DataError as e:
         raise ParseError(text(e))
     else:
-        if accepted_format == 'json':
+        if accepted_format == 'json' or not accepted_format:
```

Comment:
Why do we need `or not accepted_format`?

Reply:
Sometimes the `format` kwarg is empty; the format kwarg is the value we use to determine what type of response to return(but by default we return JSON)